llama_stack/apis/common/content_types.py

```diff
@@ -27,6 +28,9 @@ def validator(cls, values):
             return values
         return {"url": values}
 
+    class Config:
```

Comment:
should we only do this if data, not URL?

Reply:
URL should already be serializable (e.g. string1 test passes already).

Reply:
where doe sit specify that its only for data and not url?

Reply:
The key is a type that has custom serialization (e.g. here the key is "bytes"). 
```
json_encoders = {bytes: lambda v: base64.b64encode(v).decode("utf-8")}
```

Reply:
actually just found out there's another way for custom serialization that specifically targets the `data` field.  https://docs.pydantic.dev/latest/concepts/serialization/#custom-serializers